lxd/storage/pool_interface.go

```diff
@@ -126,6 +126,7 @@ type Pool interface {
 	UnmountCustomVolume(projectName string, volName string, op *operations.Operation) (bool, error)
 	ImportCustomVolume(projectName string, poolVol *backupConfig.Config, op *operations.Operation) (revert.Hook, error)
 	RefreshCustomVolume(projectName string, srcProjectName string, volName, desc string, config map[string]string, srcPoolName, srcVolName string, snapshots bool, op *operations.Operation) error
+	UpdateCustomVolumeBackupFile(projectName string, volName string, snapshots bool, instances []instance.Instance, op *operations.Operation) error
```

Comment:
As this is primarily operating on Instance Backup Files, lets put the Instances front and center (especially as custom volumes dont have backup files).

Suggest a name like `UpdateInstanceBackupFilesCustomVolume` - i know its a bit of a mouthful, but seems clearer than the proposed.